dashboard/views.py

```diff
@@ -69,4 +70,4 @@ def _find_metric_or_404(slug):
             return MC.objects.get(slug=slug)
         except MC.DoesNotExist:
             continue
-    raise Http404(f"Could not find metric with slug {slug}")
+    raise Http404(_(f"Could not find metric with slug {slug}"))
```

Comment:
I don't think this will work, since the f-string will be evaluated before `gettext()` is called. I don't think we can use f-strings here, we need to use the old `%` or `str.format`:
```suggestion
    raise Http404(_("Could not find metric with slug %s") % slug)
```

Come to think of it, this kind of thing should have been caught by our linter, do you know if there might be some additional rules/plugins we could install that would have caught this?

Reply:
Yes good catch - since I did this I've actually read up on f strings and gettext. I'm not sure about linters catching this, but that would be nice to have.